sky/shell/platform/mojo/main_mojo.cc

```diff
@@ -16,12 +16,14 @@
 #include "sky/engine/public/platform/sky_settings.h"
 #include "sky/shell/shell.h"
 #include "sky/shell/platform/mojo/content_handler_impl.h"
+#include "sky/shell/platform/mojo/dart_tracing.h"
 
 namespace sky {
 namespace shell {
 namespace {
 
 const char kEnableCheckedMode[] = "--enable-checked-mode";
+const char kVmCompleteTimeline[] = "--vm-complete-timeline";
```

Comment:
Flutter already has --trace-startup which will give you the complete timeline (minus events you don't care about).